dockers/docker-dhcp-server/lease_update.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pid=`ps aux | grep 'dhcpservd.py' | grep -nv 'grep' | awk '{print $2}'`
+kill -s 10 ${pid}
```

Comment:
need error handling for process not exists

Reply:
Fixed